src/Cli/dotnet/commands/dotnet-workload/install/WorkloadInstallCommand.cs

```diff
@@ -175,7 +175,12 @@ public void InstallWorkloads(IEnumerable<WorkloadId> workloadIds, bool skipManif
             {
                 // Update currently installed workloads
                 var installedWorkloads = _workloadInstaller.GetWorkloadInstallationRecordRepository().GetInstalledWorkloads(_sdkFeatureBand);
-                workloadIds = workloadIds.Concat(installedWorkloads).Distinct();
+                var previouslyInstalledWorkloads = installedWorkloads.Intersect(workloadIds);
+                if (previouslyInstalledWorkloads.Any())
+                {
+                    _reporter.WriteLine(string.Format(LocalizableStrings.WorkloadAlreadyInstalled, string.Join(" ", previouslyInstalledWorkloads)).Yellow());
```

Comment:
I think we had some encoding issues with Yellow messages not too long ago, is this doing things correctly?  Was the issue before that we were using `Console` instead of the reporter?

Reply:
Yes, I believe that issue was console vs reporter, I'm not aware of any issues with .Yellow()